Formula/haste-client.rb

```diff
@@ -26,20 +26,29 @@ class HasteClient < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "f58ba7733cf6b73c7ad7be755d4d31c88af9577a43f9cd4658d8732bdd8bad3b"
   end
 
-  depends_on "ruby" if MacOS.version <= :sierra
```

Comment:
Why doesn't
```ruby
uses_from_macos "ruby", since: :high_sierra
```
work here again?

Reply:
It probably does, I just elected to not make the change in this PR. Would doing this require new bottles?

Reply:
Oh, the whole point us that
```ruby
uses_from_macos "ruby", since: :high_sierra
```
should be equivalent to
```ruby
depends_on "ruby" if MacOS.version <= :sierra
```